Subject: use AC_CHECK_HEADER instead of AC_CHECK_FILE
From: Michael Olbrich <m.olbrich@pengutronix.de>

AC_CHECK_FILE is not possible when cross-compiling. Use AC_CHECK_HEADER
instead. It's the correct way to check for header files.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 configure.ac |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -84,14 +84,14 @@ sdkdir=$(pkg-config --variable=sdkdir xo
 AC_HEADER_STDC
 
 if test "$DRI" != no; then
-	AC_CHECK_FILE([${sdkdir}/dri.h],
+	AC_CHECK_HEADER([${sdkdir}/dri.h],
                       [have_dri_h="yes"], [have_dri_h="no"])
-	AC_CHECK_FILE([${sdkdir}/sarea.h],
+	AC_CHECK_HEADER([${sdkdir}/sarea.h],
                       [have_sarea_h="yes"], [have_sarea_h="no"])
-	AC_CHECK_FILE([${sdkdir}/dristruct.h],
+	AC_CHECK_HEADER([${sdkdir}/dristruct.h],
                       [have_dristruct_h="yes"], [have_dristruct_h="no"])
-	AC_CHECK_FILE([${sdkdir}/damage.h],
-                      [have_damage_h="yes"], [have_damage_h="no"])
+	AC_CHECK_HEADER([${sdkdir}/damage.h],
+                      [have_damage_h="yes"], [have_damage_h="no"],[${sdkdir}/region.h])
 fi
 
 AC_MSG_CHECKING([whether to include DRI support])
